docs/developer/concepts/duck-typing.md

```diff
@@ -26,15 +38,15 @@ spec:
     name: pointee
 ```
 
-Assume the expected shape of a "Sizable" duck type is that in the status, we
-expect the following schema shape:
+If the expected shape of a Sizable duck type is that in the status, the schema shape is the
+following: <!-- If possible, we want to call these examples "resource" shapes, not "schema" shapes, because we've only talked of "resources" so far. Ask an engineer if that's okay. -->
```

Comment:
As discussed on Slack, if we're talking about the duck type's shape we want to talk about the *schema*, *schema shape* or *resource's schema*, but not *resource shape*.

Reply:
Thanks, I'll leave things as they are.